test/ui-test/testSuites/suite_status/tst_communityFlows/test.feature

```diff
@@ -34,21 +34,23 @@ Feature: Status Desktop community
     Scenario Outline: Admin creates a community channel
```

Comment:
Some identation issues in the previous examples (User creates a community) scenario